x/ethbridge/genesis.go

```diff
@@ -0,0 +1,37 @@
+package ethbridge
+
+import (
+	"github.com/Sifchain/sifnode/x/ethbridge/types"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	abci "github.com/tendermint/tendermint/abci/types"
+)
+
+// InitGenesis import genesis data from GenesisState
+func InitGenesis(ctx sdk.Context, keeper Keeper, data types.GenesisState) (res []abci.ValidatorUpdate) {
+
+	for _, token := range data.PeggyTokens {
```

Comment:
My understanding was that we weren't migrating data from genesis, but rather from keepers that were already live and storing data in our system. Does this code address this?

Reply:
the code just for 42 upgrade. export current data in keeper to genesis file and then new version sifchain can read the data from it. export command and sample genesis file put in the docs folder.